english/movie-ripoffs/matrix.md

```diff
@@ -0,0 +1,31 @@
+Nothingness... Endless drifting through the void...
+
+Dim red light. The taste of oil. The smell of sweaty feet. The feel of wet nappies. ...
+
+You regain consciousness on a stretcher in some type of underground sewer, surrounded by a bunch of ragged hobos with mad staring eyes.
+
+"He's awake, Morpheus." says a dark haired waif on your left. 
+
+A large leering, gap-toothed man steps over and looks into your face.  "Time to learn Krav Maga." he says as he jams something sharp into the back of your skull.
+
+A split second later you realise you're in the middle of a brightly lit japanese dojo, garbed in oriental combat clothes.
+The same large toothy man, presumably Morpheus, stands in front of you, giggling uncontrollably.
+
+"What's going on?" you ask in puzzled trepidation as you watch him slowly crouch into a fighting pose.
+
+"That's for me to know and you to find out" Morpheus sniggers, lashing his foot into your groin, 
+and immediately following up with a flurry of kicks, punches and head butts. 
+
+"What the hell?" you yell in pain and consternation as you grovel in the fetal position, trying to fend off the unwarranted attack.
+
+"Do you think this is AIR that you're breathing?" Morpheus suggests cryptically as he slaps you across the face.
+"Actually it's not!", he giggles, "This is MY simulation, so I replaced it with Nitrous Oxide - just for laughs!"
+
+A flash of inspiration hits you.  "Well, if normal rules of physics don't apply then I can just use magic." you exclaim.
+
+"Accio magic wand", you yell, and a magic wand flies into your hands.   "Expectorate!", you shout, and a blast of gooey liquid smashes
+Morpheus across the room.
+
+Choose the next spell:
+["Apparate" to get the hell out of here](./hogwarts.md)
+["Avada Kedavra" to finish Morpheus off](./matrix-boss.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.